astropy/utils/masked/core.py

```diff
@@ -954,6 +954,10 @@ def max(self, axis=None, out=None, **kwargs):
             axis=axis, out=out, **self._reduce_defaults(kwargs, np.nanmin)
         )
 
+    # handy aliases for standard numpy extrema functions:
+    amax = max
```

Comment:
why just use `np.min` and `np.max`?

Reply:
Please remove those. `np.ndarray` also does not have `.amin` and `.amax`.

Reply:
Moving this to a different PR/effort.